packages/core/chrome/core-chrome-browser-internal/src/ui/loading_indicator.scss

```diff
@@ -1,9 +0,0 @@
-.kbnLoadingIndicator-hidden {
-  visibility: hidden;
-  animation-play-state: paused;
-}
-
-.euiHeaderSectionItem .euiButtonEmpty__text {
-  // stop global header buttons from jumping during loading state
-  display: flex;
```

Comment:
I did not port these styles over, because I do not see the global header buttons jumping around during loading state.

Reply:
I just added this in a recent PR. See if you notice it when throttling your browser... in particular, the spaces button (loading rectangle) would jump up and right in a flash.